Momento/ScsDataClient.cs

```diff
@@ -5,33 +5,34 @@
 using CacheClient;
 using Google.Protobuf;
 using Grpc.Core;
+using System.Collections.Generic;
 
 namespace MomentoSdk
 {
     internal sealed class ScsDataClient : IDisposable
     {
         private readonly DataGrpcManager grpcManager;
         private readonly uint defaultTtlSeconds;
-        private readonly uint dataClientOperationTimeoutSeconds;
-        private const uint DEFAULT_DEADLINE_SECONDS = 5;
+        private readonly uint dataClientOperationTimeoutMilliseconds;
```

Comment:
just curious why switching to milli?

Reply:
We thought that more control over request timeout might be needed since we're offering a performant cache product. Also matching with the Python SDK as well as making the failure test possible to make sure multi get accept failed responses as argument.
cc @eaddingtonwhite